src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -646,45 +644,6 @@ private String fillUpInstructorJoinFragment(InstructorAttributes instructor, Str
     }
 
     /**
-     * Generates the system error report email for the given {@code error}.
-     */
-    public EmailWrapper generateSystemErrorEmail(
-            String requestMethod, String requestUserAgent, String requestPath, String requestUrl,
-            String requestParams, UserType userType, Throwable error) {
-
-        String errorMessage = error.getMessage();
-        String stackTrace = TeammatesException.toStringWithStackTrace(error);
-
-        // If the error doesn't contain a short description, retrieve the first line of stack trace.
-        // truncate stack trace at first "at" string
-        if (errorMessage == null) {
-            int msgTruncateIndex = stackTrace.indexOf("at");
-            if (msgTruncateIndex > 0) {
-                errorMessage = stackTrace.substring(0, msgTruncateIndex).trim();
-            } else {
-                errorMessage = "";
-            }
-        }
-
-        String actualUser = userType == null || userType.id == null ? "Not logged in" : userType.id;
-
-        String emailBody = Templates.populateTemplate(EmailTemplates.SYSTEM_ERROR,
-                "${actualUser}", actualUser,
-                "${requestMethod}", requestMethod,
-                "${requestUserAgent}", requestUserAgent,
-                "${requestUrl}", requestUrl,
-                "${requestPath}", requestPath,
-                "${requestParameters}", requestParams,
-                "${errorMessage}", errorMessage,
-                "${stackTrace}", stackTrace);
-
-        EmailWrapper email = getEmptyEmailAddressedToEmail(Config.SUPPORT_EMAIL);
-        email.setSubject(String.format(EmailType.ADMIN_SYSTEM_ERROR.getSubject(), Config.getAppVersion(), errorMessage));
```

Comment:
Looks like I missed this. This enum `EmailType.ADMIN_SYSTEM_ERROR` should be fine to be removed as well.